addon/components/frost-table-row.js

```diff
@@ -65,6 +65,11 @@ export default Component.extend({
 
   // == DOM Events ============================================================
 
+  click (event) {
+    event.stopPropagation()
+    this.$().find('.frost-table-row-selection').trigger('click')
```

Comment:
Are we doing an action on click for the table on when we click on a button or something like that (similar to list)? If we are using a button to do an action based on selection then we should use a similar implementation to the list and I'm not seeing anything like this in the list.